Dockerfile

```diff
@@ -41,6 +41,11 @@ RUN \
         curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/linux_64bit/session-manager-plugin.rpm" -o "session-manager-plugin.rpm" &&\
         yum install -y session-manager-plugin.rpm
 
+# Add user to /etc/passwd for keygen in Makefile
+ARG USER
+RUN \
+        echo "user:x:${USER}:0:root:/:/bin/bash" >> /etc/passwd
```

Comment:
Maybe use the `useradd` command instead? That would feel less error prone to me.

Reply:
@reweeden I've been playing with using `useradd` and it keeps hanging at the `exporting layers` part of the image build.  Have you seen this before?  

```
RUN \
         useradd -M -u ${USER} user
```